AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthorizationProviderAdapter+SignedInSession.swift

```diff
@@ -222,4 +220,18 @@ extension AuthorizationProviderAdapter {
                                                 cognitoTokensResult: tokenResult)
         completionHandler(.success(authSession))
     }
+
+    /// Check if the error is related to improper configuration of Cognito Identity Pool
+    private func identityPoolConfigured(error: Error) -> Bool {
```

Comment:
1. See note above re method name
1. I don't see this using any instance members, so make this `static`

More importantly, it's hard to wade through the double negatives in this method logic: the method comment asserts that we're looking to see if the error is related to improper configuration of an identity pool, but we return `false` if we find that the error is in fact a `cognitoIdentityPoolNotConfigured` error. How is that error not related to improper configuration? Why are just these two error conditions sufficient to assert that the error isn't related to improper configuration?